static/src/javascripts/bootstraps/commercial.ts

```diff
@@ -60,17 +60,27 @@ if (!commercialFeatures.adFree) {
 	);
 }
 
-const loadHostedBundle = () => {
+const loadHostedBundle = (): Promise<void> => {
 	if (config.get('page.isHosted')) {
 		return new Promise((resolve) => {
 			require.ensure(
 				[],
 				(require) => {
-					const hostedAbout = require('commercial/modules/hosted/about');
-					const initHostedVideo = require('commercial/modules/hosted/video');
-					const hostedGallery = require('commercial/modules/hosted/gallery');
-					const initHostedCarousel = require('commercial/modules/hosted/onward-journey-carousel');
-					const loadOnwardComponent = require('commercial/modules/hosted/onward');
+					/* eslint-disable
+					 	@typescript-eslint/no-var-requires,
+						@typescript-eslint/consistent-type-imports,
+						--
+						these are chunked by webpack */
+					const hostedAbout = require('commercial/modules/hosted/about') as typeof import('commercial/modules/hosted/about');
```

Comment:
Not sure about this but is the synchronous equivalent of doing

```suggestion
					const hostedAbout = await import('commercial/modules/hosted/about');
```

Reply:
Yes, I think so. `require.ensure` does not allow for an async function as a second argument, sadly.

Reply:
Gotcha